File: js/ticket.js
class Ticket {
    constructor(id = null, usuario = null, fechaInicio = null, fechaFin = null, pagado = null, fechaPago = null, importe = null) {
        this._id = id;
        this._usuario = usuario;
        this._fechaInicio = fechaInicio;
        this._fechaFin = fechaFin;
        this._pagado = pagado;
        this._fechaPago = fechaPago;
        this._importe = this._importe;
    }
    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get usuario() {
        return this._usuario;
    }

    set usuario(usuario) {
        this._usuario = usuario;
    }

    get fechaInicio() {
        return this._fechaInicio;
    }

    set fechaInicio(fechaInicio) {
        this._fechaInicio = fechaInicio;
    }

    get fechaFin() {
        return this._fechaFin;
    }

    set fechaFin(fechaFin) {
        this._fechaFin = fechaFin;
    }

    get pagado() {
        return this._pagado;
    }

    set pagado(pagado) {
        this._pagado = pagado;
    }

    get fechaPago() {
        return this._fechaPago;
    }

    set fechaPago(fechaPago) {
        this._fechaPago = fechaPago;
    }

    get importe() {
        return this._importe;
    }

    set importe(importe) {
        this._importe = importe;
    }
}

File: App_Code/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;
using System.Xml.Serialization;

namespace es.itsduero.iecscyl
{
    [DataContract]
    public class Ticket
    {
        [XmlElement ("TicketID")]
        public int ID { get; set; }
        [XmlElement("TicketUsuario")]
        public Usuario usuario { get; set; }
        [XmlElement("TicketFechaInicio")]
        public DateTime fechaInicio { get; set; }
        [XmlElement("TicketFechaFin")]
        public DateTime fechaFin { get; set; }
        [XmlElement("TicketPagado")]
        public bool pagado { get; set; }
        [XmlElement("TicketFechaPago")]
        public DateTime fechaPago { get; set; }
        [XmlElement("TicketImporte")]
        public Double importe { get; set; }
    }
}